.drone.yml

```diff
@@ -0,0 +1,49 @@
+---
+kind: pipeline
+type: docker
+name: review
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: vet
+  image: golang:1.14
+  commands:
+  - go vet ./...
+  volumes:
+  - name: deps
+    path: /go/pkg
```

Comment:
we dont need this step. The linter handles go vet

Reply:
why is it being setup on 1.14? is there a reason we can't use the latest version?